.ci/pipeline_definitions

```diff
@@ -0,0 +1,48 @@
+machine-controller-manager-provider-equinix-metal:
+  template: 'default'
+  base_definition:
+    repo: ~
+    traits:
+      version:
+        preprocess:
+          'inject-commit-hash'
+        inject_effective_version: true
+      publish:
+        dockerimages:
+          machine-controller-manager-provider-equinix-metal:
+            inputs:
+              repos:
+                source: ~ # default
+              steps:
+                build: ~
+            registry: 'gcr-readwrite'
+            image: 'eu.gcr.io/gardener-project/gardener/machine-controller-manager-provider-equinix-metal'
+    steps:
+      check:
+        image: 'golang:1.13.5'
+      build:
+        image: 'golang:1.13.5'
+        output_dir: 'binary'
+      test:
+        image: 'golang:1.13.5'
```

Comment:
```suggestion
      check:
        image: 'golang:1.16.2'
      build:
        image: 'golang:1.16.2'
        output_dir: 'binary'
      test:
        image: 'golang:1.16.2'
```
Probably move the images to [recent golang](https://hub.docker.com/layers/golang/library/golang/1.16.2/images/sha256-d24c4fa66898a1ab5b7aa22ad9a3ed61210d7a1f8af56c684d12b750e5ae7965?context=explore) version.